classification/dataset.py

```diff
@@ -328,45 +334,43 @@ def get_num_classes(self) -> int:
         return self.num_classes
 
 
-def get_datasets(
-        transforms = None, CONFIG=None
-        ):
+def get_datasets(transforms = None):
     """ Returns train and validation datasets, does random sampling for train/valid split, adds transforms to dataset
     """
 
 
-    train_p = CONFIG.train_test_split
-    path = CONFIG.dataframe
+    train_p = cfg.train_test_split
+    path = cfg.dataframe_csv
     # Load the dataset
     data = pd.read_csv(path, usecols = [
-        CONFIG.file_name_col,
-        CONFIG.manual_id_col,
-        CONFIG.offset_col,
-        CONFIG.duration_col
+        cfg.file_name_col,
+        cfg.manual_id_col,
+        cfg.offset_col,
+        cfg.duration_col
     ], dtype={
-        CONFIG.file_name_col: str,
-        CONFIG.manual_id_col: str,
-        CONFIG.offset_col: float,
-        CONFIG.duration_col: float})
+        cfg.file_name_col: str,
+        cfg.manual_id_col: str,
+        cfg.offset_col: float,
+        cfg.duration_col: float})
     
     #for each species, get a random sample of files for train/valid split
-    train_files = data.groupby(CONFIG.manual_id_col, as_index=False).apply(
-        lambda x: pd.Series(x[CONFIG.file_name_col].unique()).sample(frac=train_p)
+    train_files = data.groupby(cfg.manual_id_col, as_index=False).apply(
```

Comment:
This does not work if training on a single species since groupby outputs a different format if there's only one group BUT apparently this is not in the scope of the project. You can replace it with the following to make it generalizable to that case.
```
   unique_files = data.groupby(cfg.file_name_col, group_keys=False).apply(lambda df: df.sample(1)) # unique
    train_files = unique_files.groupby(cfg.manual_id_col, group_keys=False).apply(lambda df: df.sample(frac=train_p))
    train = data[data["FILE NAME"].isin(train_files["FILE NAME"])]
    valid = data[~data["FILE NAME"].isin(train_files["FILE NAME"])]
```